### PR TITLE
Enable releasing to jcenter

### PR DESCRIPTION
**Pull request by kryptidikettu**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,19 +1,36 @@
-apply plugin: 'com.android.library'
-
-group = 'org.slamon.android'
```
Comment (jjonek) [inline build.gradle:3]: actually, could the group be just _org.slamon_, as it's in the java lib? or would it conflict somehow?

Comment (kryptidikettu) [inline build.gradle:3]: Actually, I'm not sure. It used to be 'org.slamon.android' but I don't know how it fairs if I change it to just 'org.slamon'.

**Comment by kryptidikettu** [comment]
https://bintray.com/stealthyloner/android/slamon-agent/387ae2c/view#files This is the result after publishing to bintray

**Comment by kryptidikettu** [comment]
The bintray release obviously doesn't happen with travis, since the unit tests are broken. I'm trying to fix those before merge

**Pull request merged**